app/assets/javascripts/discourse/models/category.js.es6

```diff
@@ -65,6 +65,16 @@ const Category = RestModel.extend({
     return (parentLevel || -1) + 1;
   },
 
+  @discourseComputed("subcategories")
+  isGrandParent(subcategories) {
+    return (
+      subcategories &&
+      subcategories.some(
```

Comment:
Does `some` work in IE11?

Reply:
[Yes.](https://caniuse.com/#search=some)